storage/persistent_storage/persistent-storage-aws.adoc

```diff
@@ -6,24 +6,50 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+ifdef::openshift-rosa,openshift-dedicated[]
+{product-title} (ROSA) clusters are prebuilt with four storage classes that use AWS Elastic Block Store (EBS) volumes. These storage classes are ready to use and some familiarity with Kubernetes and AWS is assumed.
+
+Following are the four prebuilt storage classes:
```

Comment:
I filed one jira(https://issues.redhat.com/browse/SDA-7296) for prebuilt storage classes, if that is accepted, doc may need to be updated as well.
cc @yingzhanredhat

Reply:
Thanks for the heads-up!